### DISCUSSION TITLE
Issue with Retrieving "Description" Field from IFC to GLB Conversion in Three.js

### DISCUSSION THREAD

**Discussion by tony-huaulme** [IfcOpenShell/IfcOpenShell#5819] [General]
Hi everyone,

I’m working on converting an IFC file to a GLB model, and I’ve run into a problem. While the conversion works well, I cannot seem to find the "Description" field that I know exists in my IFC file when inspecting the resulting GLB model in Three.js.

Here’s a screenshot of the original IFC file, which contains the "Description" field:
![image](https://github.com/user-attachments/assets/85ae9feb-0aae-45a7-b2c8-466c25eb0316)

However, when I load the .glb model in Three.js, the "Description" field seems to be missing, even though the model contains other metadata like the name, GUID, etc. Here’s a snapshot of the data I get after loading the .glb file:
![image](https://github.com/user-attachments/assets/8e56e584-edae-428c-a309-9dbec05b7d86)
I’ve also noticed that the "name" field in the model appears to be a concatenation of "IfcEntity" and "Name" of the element.

**### What I need:**

I need to retrieve the "Description" set in the original IFC file after it’s converted to GLB. I’m using [IfcOpenShell](https://github.com/IfcOpenShell) for the conversion process, and it’s been working great for other aspects, but I can’t seem to get this metadata in the final model.

Would anyone be able to provide some guidance or suggestions on how to ensure the "Description" field is properly retained and accessible in the GLB model when using Three.js?

Thanks so much for your amazing work on IfcOpenShell, and I hope someone can help push this project to the next level!

**Comment by aothms** [comment]
The typical workflow people use is to export the meta-data to .xml also using IfcConvert. The .glb file really only contains the geometry and an identifier to link between the .glb and the .xml when both are exported. Have a look at for ex. https://github.com/aecgeeks/ifc-pipeline where both are used side by side.

**Reply by tony-huaulme** [reply]
Thank you a lot !
